src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+
+}
+
+.stars__star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
move to stars_start

```
background-repeat: no-repeat;
background-position: center;
```

Reply:
?

Reply:
But, I don't understand what I have to do with it, Like what means stars__start?

Reply:
> ?


But, I don't understand what I have to do with it, Like what means stars__start?

Reply:
it's not working when I remove
background-repeat: no-repeat;
background-position: center;